src/main/java/com/coveros/selenified/Capabilities.java

```diff
@@ -186,7 +189,7 @@ public void setupProxy() {
      * Additionally, the iedriverVersion is set to match the selenium version as suggested, if ie is the chosen browser
      * Finally, the default platform for edge is set to windows 10
      */
-    public void setupSauceCapabilities() {
+    public void setupSauceCapabilities() throws InvalidHubException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'com.coveros.selenified.exceptions.InvalidHubException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)